mlflow/recipes/resources/recipe_dag_template.html

```diff
@@ -134,7 +134,7 @@
     <script type="text/javascript">
       require.config({
         paths: {
-            "mermaid": "https://cdn.jsdelivr.net/npm/mermaid/dist/mermaid.min",
+            "mermaid": "https://cdn.jsdelivr.net/npm/mermaid@9.3.0/dist/mermaid.min",
```

Comment:
I assume this is the latest version that doesn't break our code?

Reply:
Yes correct. This is the 1 before the latest which is 9.4.0